### PR TITLE
Create new translations section in correct-mistakes app

### PR DESCRIPTION
**Pull request by easternbloc**
This is the files that will be built with hof-transpiler into a default locales

DSP-225 :ok_hand:

### CODE REVIEW

File: apps/correct-mistakes/translations/src/en/buttons.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "test": "Start",
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/buttons.json]: hmmm?

File: apps/correct-mistakes/translations/src/en/buttons.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "continue": "Continue",
+  "submit": "send",
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/buttons.json]: hmmm?

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+    "options": {
+      "post-office": {
+        "label": "Post office"
+      },
+      "sponsor": {
+        "label": "Sponsor"
+      }
+    }
+  },
+  "arrange-collection-radio": {
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/fields.json]: I don't think this is necessary for /correct-mistakes

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+  "delivery-date-day": {
+    "label": "Day"
+  },
+  "delivery-date-month": {
+    "label": "Month"
+  },
+  "delivery-date-year": {
+    "label": "Year"
+  },
+  "collection-date": {
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/fields.json]: None of the collection translations are necessary for /correct-mistakes

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 5]
```diff
@@ -0,0 +1,470 @@
...
+      },
+      "change-person": {
+        "label": "Change the person I requested to collect my BRP"
+      },
+      "cancel-request": {
+        "label": "Cancel request for someone else to collect my BRP"
+      }
+    }
+  },
+  "someone-else-fullname": {
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/fields.json]: None of the somone-else translations are necessary for /correct-mistakes

File: apps/correct-mistakes/translations/src/en/fields.json [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+  "change-person-date-year": {
+    "label": "Year"
+  },
+  "change-person-nationality": {
+    "label":  "Country of nationality"
+  },
+  "change-person-id-number": {
+    "label": "Identity document number <br/>(This could be UK/EU driving licence, passport or BRP)"
+  },
+  "reason-radio": {
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/fields.json]: None of the reason translations are necessary for /correct-mistakes

**Comment by JoeChapman** [comment]
There is a lot of content that is not necessary for the /correct-mistakes app

**Comment by JoeChapman** [comment]
Also, where is the JIRA ticket?

**Comment by easternbloc** [comment]
Sorry this was just the initial move translations file. You make a good point lots of stuff needs deleting.

File: apps/correct-mistakes/translations/src/en/buttons.json (hunk 2 above)
Comment (easternbloc) [inline apps/correct-mistakes/translations/src/en/buttons.json]: Could you explain what the hmm is for is it the lower case s?

Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/buttons.json]: Sorry, I should be more specific: It's for the `"test": "Start"`

**Comment by easternbloc** [comment]
Updated with lots of deletions :)

**Comment by easternbloc** [comment]
Updated with latest version of hof-transpiler.

**Comment by JoeChapman** [comment]
@easternbloc the tests are failing

File: package.json [hunk 7]
```diff
@@ -60,7 +60,7 @@
     "eslint-plugin-filenames": "^0.1.1",
     "eslint-plugin-mocha": "^0.2.2",
     "eslint-plugin-one-variable-per-var": "0.0.3",
-    "hof-transpiler": "0.0.1",
+    "hof-transpiler": "0.0.2",
```
Comment (JoeChapman) [inline package.json]: maybe this should be a devDependency

**Comment by JoeChapman** [comment]
You need to update travis.yml to build the translations before running the tests

**Comment by easternbloc** [comment]
I think i'm gonna do it as a post install seems to make more sense.

File: apps/correct-mistakes/translations/src/en/buttons.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
+  "continue": "Continue",
+  "submit": "Submit",
```
Comment (JoeChapman) [inline apps/correct-mistakes/translations/src/en/buttons.json]: this needs to be `"send": "Send"`

Comment (easternbloc) [inline apps/correct-mistakes/translations/src/en/buttons.json]: Even worse... "send"

**Comment by JoeChapman** [comment]
Whoop de doo

**Pull request merged**